### ISSUE TITLE
my_collect_spec.rb Issue

### ISSUE THREAD

**Issue by yieldjessyield** [learn-co-curriculum/my-collect#18]
In the my_collect_spec.rb file, on line 40 (in the "returns a new collection of appropriately modified elements, in this case student first names" section) it says:

```
expect(my_collect(students) do |language|
```

I think it should be changed to:

```
expect(my_collect(students) do |student|
```

Otherwise "student" is undefined in the rest of the code (lines 41-43):

```
student.split(" ").first
    end).to eq(["Tim", "Tom", "Sophie", "Antoin"])
  end
```

**Comment by jsegarra3** [comment]
I also had this issue and had to modify the spec to make the test pass. I second this change.

**Comment by citurbe** [comment]
I third!

**Comment by gj** [comment]
Fixed in #17. Thanks @yieldjessyield !
